bin/create-migration

```diff
@@ -16,8 +16,8 @@ directories.each do |directory|
     exit 2
   end
 
-  if File.basename(directory) == 'foreman.migrations'
-    # Used by the foreman scenario
+  if ['foreman.migrations', 'katello.migrations'].include?(File.basename(directory))
```

Comment:
I don't believe this change is needed. Katello and Foreman scenarios currently use different timestamp formats for their migrations and hence why this code exists as it does (see the else block).

Reply:
Remvoed